bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -41,10 +41,11 @@ class NominationContextModal(discord.ui.Modal, title="New Nomination"):
         max_length=REASON_MAX_CHARS - 110
     )
 
-    def __init__(self, api: NominationAPI, message: discord.Message, noms_channel: discord.TextChannel):
+    def __init__(self, api: NominationAPI, message: discord.Message, noms_channel: discord.TextChannel, bot: Bot):
```

Comment:
Since it's taking the `api`, `bot`, and calling methods on the cog, it may be cleaner to make it take the cog itself as an argument instead (and not have `maybe_relay_update` as a staticmethod).

I don't really mind though.